modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Tabs.js

```diff
@@ -4,14 +4,20 @@
  */
 
 import ClayTabs from '@clayui/tabs';
-import React, {useEffect, useRef} from 'react';
+import React, {useEffect, useRef, useState} from 'react';
 
-const Panel = ({children}) => {
+const Panel = ({children: tabPanel}) => {
 	const ref = useRef();
 
 	useEffect(() => {
-		ref.current.appendChild(children);
-	}, [children]);
+		const fragment = document.createDocumentFragment();
+
+		while (tabPanel.firstChild) {
+			fragment.appendChild(tabPanel.firstChild);
+		}
+
+		ref.current.appendChild(fragment);
+	}, [tabPanel]);
```

Comment:
Why this direct DOM compexity? Can we avoid `useEffect` and just do something like 
```js
return <>{children}</>;
```

Reply:
this comes from the original impl of the clay tabs tag. The thing is that `{children}` here is not a list of React.element but just DOM elements representing the children of the tabs as they come from the SSR. We may review this in subsequent tasks as I'm not extremely happy with this duality

Reply:
Yeah, I tried exactly what you suggest @markocikos and found out what @dsanz explained

Reply:
Got it. Yeah, this makes sense then, unless we want to refactor upstream code :+1: